vnext/src/Libraries/Components/Touchable/TouchableHighlight.windows.js

```diff
@@ -330,11 +330,6 @@ class TouchableHighlight extends React.Component<Props, State> {
         accessibilityPosInSet={this.props.accessibilityPosInSet} // [Windows]
         accessibilitySetSize={this.props.accessibilitySetSize} // [Windows]
         onAccessibilityTap={this.props.onAccessibilityTap} // [Windows]
-        acceptsKeyboardFocus={
```

Comment:
I thought we removed this in the PR adding focusable support... Having both this and focusable set is super jank. Right now we can send both props with different values and the one set is dependent on ordering. The case where they conflict is when a button is disabled, or doesn't have an event handler. I think acceptsKeyboardFocus's value would win, which means this adds a small behavior change bringing UWP closer to Android behavior.